editors/sc-ide/core/main.cpp

```diff
@@ -162,8 +163,9 @@ bool SingleInstanceGuard::tryConnect(QStringList const & arguments)
     mIpcServer = new QTcpServer(this);
     bool listening = mIpcServer->listen(QHostAddress(QHostAddress::LocalHost), SingleInstanceGuard::Port);
     if (listening) {
-        mIpcChannel = new ScIpcChannel(mIpcServer->nextPendingConnection(), QString("SingleInstanceGuard"), this);
-
+		// socket is NULL here in Windows, so we pass it in onNewIpcConnection()
```

Comment:
Yes, that's what I thought, but then I am surprised that this works in other platforms. Unless they are using another mechanism. So I assumed that was the case.